### ISSUE TITLE
Ensure data types in Model Editor

### ISSUE THREAD

**Issue by jvanhoefer** [yaml2sbml-dev/yaml2sbml#100]
When exporting the current FSP example to yaml instead of sbml, one gets quite weird state initial values. Would be nice to ensure & type cast the data types in all parts of the Model Editor. (Have to think on how this is best done...)
```yaml
    - stateId: x_0_13
      rightHandSide: -(k_1 + (k_2 + k_3)*0 + k_4*13) * x_0_13 + k_2 * 1 * x_1_13+
            k_3 * 0 * x_0_12+ k_4 * 14 * x_0_14
      initialValue: !!python/object/apply:numpy.core.multiarray.scalar
      - *id001
      - !!binary |
            DAmnXBiq4j4=
```

**Comment by jvanhoefer** [comment]
Also kinda related to #99...

**Comment by jvanhoefer** [comment]
maybe use Pydantic here, also related to #107 

Furthermore it might be a good idea to check, if the different libsbml calls are successful in `yaml2sbml` (e.g. `setId()` returns a corresponding flag...)

**Comment by jvanhoefer** [comment]
At least the issue with the FSP example is resolved in #117
